src/common/snippets/include/snippets/lowered/pass/iter_handler.hpp

```diff
@@ -81,22 +81,18 @@ class SetEvaluateOnce : public snippets::lowered::pass::RangedPass {
 };
 
 /**
- * @interface SetBrgemmBeta
- * @brief The pass updates all CPUBrgemm nodes with a new beta value
- * @param m_beta - beta which must be set
+ * @interface DummyPass
+ * @brief The empty pass. For example, It can be used to force insertion of first specific iteration
  * @ingroup snippets
  */
-class SetBrgemmBeta : public snippets::lowered::pass::RangedPass {
+class DummyPass : public snippets::lowered::pass::RangedPass {
```

Comment:
Just an idea: maybe it is worth to move this `DummyPass` to `BrgemmCPU` node? I think it is reasonable since this pass is used as a workaround only for one specific node. Ideally, the pass should be placed in private section, but in this case I don't now how to reuse it in tests for reference LIR composition, to be honest

Reply:
As I already did for `SetBrgemmBeta` for TPP 🤔  Sure! Good idea! Moved.
To have opportunity to call these passes in unit tests, I decided to declare them in public sections of the corresponding tests.
Thanks!